sources_test.go

```diff
@@ -97,6 +97,40 @@ func TestFile(t *testing.T) {
 	}
 }
 
+func TestFindFilesMultipleFiles(t *testing.T) {
+	t.Parallel()
```

Comment:
Instead of three different tests, could you write this as a single table-driven test? Have a look at [TestExec](https://github.com/bitfield/script/blob/master/sources_test.go#L45) for an example. This avoids repeating the same test code, and makes it easier to add new test cases if we need to.